packages/workbox-strategies/CacheOnly.mjs

```diff
@@ -123,7 +127,10 @@ class CacheOnly {
       logger.groupEnd();
     }
 
-    return response;
+    if (response) {
```

Comment:
Nit: to me it'd make slightly more sense to invert this condition, i.e. throw when there's not a response and default to returning the response. As it is now, it looks like throwing is the default, unconditional behavior (but maybe that's just how I read it).